packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -97,6 +101,11 @@ export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX
   /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
   const holdButtonProps = usePropsFor(HoldButton);
 
+  /* @conditional-compile-remove(PSTN-calls) */
+  const alternateCallerId = useAdapter().getState().alternateCallerId;
+  /* @conditional-compile-remove(PSTN-calls) */
+  const participantNumber = usePropsFor(ParticipantList).participants.length;
```

Comment:
ambiguous attribute name `participantNumber`
I read it and think this is a participants phone number.
Something like `participantCount` would make more sense here.